packages/security_detection_engine/kibana/security_rule/785a404b-75aa-4ffd-8be5-3334a5a544dd.json

```
@@ -34,7 +34,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 11
+        "version": 10
```

```suggestion
        "version": 12
```
This is a result of a bug found in the version locks and forked rules as being discussed/worked in https://github.com/elastic/detection-rules/issues/1949

Bumping to 12.